changelogs/current.yaml

```diff
@@ -45,5 +45,9 @@ new_features:
   change: |
     added support for :ref:`%UPSTREAM_CONNECTION_ID% <config_access_log_format_upstream_connection_id>` for the upstream connection
     identifier.
+- area: aws
```

Comment:
do you want to mention this under minor behavior change? Also can mention that region field is made optional and explicitly configured value will be preferred.

Reply:
done